### PR TITLE
Landing page and Hire us re-design

### PR DESCRIPTION
**Pull request by monisha16**
Redesign of hire-us page

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -59,3 +59,38 @@ body {
...
+    circle,
+    rgba(238, 84, 32, 1) 0%,
+    rgba(245, 245, 238, 1) 86%,
+    rgba(245, 245, 238, 1) 100%
+  );
+}
+
+.contact-us {
+  background-color: rgba(190, 186, 186, 0.4);
+  /* -webkit-backdrop-filter: blur(5px);
```
Comment (mangalprada) [inline frontend/src/index.css:75]: Should we remove these lines?

File: frontend/src/components/Tooltip/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          <button className='IconButton'>{children}</button>
+        </Tooltip.Trigger>
+        <Tooltip.Portal>
+          <Tooltip.Content
+            side={position}
+            className='rounded py-0.5 px-2 bg-[#334154] text-whiteSmoke text-xs shadow-lg TooltipContent'
+            sideOffset={5}
+          >
+            {tooltip}
+            {/* <Tooltip.Arrow className='TooltipArrow' /> */}
```
Comment (mangalprada) [inline frontend/src/components/Tooltip/index.jsx]: Please remove it.

Comment (monisha16) [inline frontend/src/components/Tooltip/index.jsx]: Remove

File: frontend/src/index.css [hunk 3]
```diff
@@ -59,3 +59,38 @@ body {
...
+  background: rgb(238, 84, 32);
+  background: radial-gradient(
+    circle,
+    rgba(238, 84, 32, 1) 0%,
+    rgba(245, 245, 238, 1) 86%,
+    rgba(245, 245, 238, 1) 100%
+  );
+}
+
+.contact-us {
```
Comment (monisha16) [inline frontend/src/index.css:73]: Remove this style

File: frontend/src/pages/LandingPage/styles.css [hunk 4]
```diff
@@ -1,8 +1,36 @@
-.container{
-    /*
+.container {
+  /*
 * Created with https://www.css-gradient.com
```
Comment (monisha16) [inline frontend/src/pages/LandingPage/styles.css]: Remove

**Review by mangalprada — COMMENTED** [review]
lgtm

**Pull request merged**